src/features/Main/Header/index.jsx

```diff
@@ -1,60 +1,46 @@
-import React, { useContext, useState } from 'react';
+import React, { useContext } from 'react';
 import { AppContext } from '@edx/frontend-platform/react';
 import { getConfig } from '@edx/frontend-platform';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faCaretDown } from '@fortawesome/free-solid-svg-icons';
-import { InstitutionContext } from 'features/Main/institutionContext';
+import { faEarthAmericas } from '@fortawesome/free-solid-svg-icons';
+import { Dropdown } from '@edx/paragon';
```

Comment:
Is this the dropdown from the Topaz style? @AuraAlba

Reply:
No, for the moment I haven't changed the styles for the Dropdown in the library because I haven't seen it in another page and the styles for this dropdown are specially to match the header